Tangem/Modules/LegacyMain/LegacyMainCoordinator.swift

```diff
@@ -27,7 +27,7 @@ class LegacyMainCoordinator: CoordinatorObject {
     @Published var legacyTokenDetailsCoordinator: LegacyTokenDetailsCoordinator? = nil
     @Published var tokenDetailsCoordinator: TokenDetailsCoordinator? = nil
     @Published var detailsCoordinator: DetailsCoordinator? = nil
-    @Published var tokenListCoordinator: TokenListCoordinator? = nil
+    @Published var tokenListCoordinator: LegacyTokenListCoordinator? = nil
```

Comment:
пригодится, когда будешь тогглом закрывать
```suggestion
    @Published var legacyTokenListCoordinator: LegacyTokenListCoordinator? = nil
```

Reply:
Переименовал, спасибо